node/src/actors/chain_manager/mining.rs

```diff
@@ -930,6 +934,11 @@ pub fn build_block(
             }
         };
 
+        if let Err(e) = validate_rad_request(&dr_tx.body.dr_output.data_request, active_wips) {
+            log::warn!("DataRequest Transaction invalid: {}", e);
+            continue;
+        }
```

Comment:
We can add a comment that says that this can be removed after the WIP-0019 is activated